src/debugAdapter/goDebug.ts

```diff
@@ -206,6 +206,7 @@ interface LaunchRequestArguments extends DebugProtocol.LaunchRequestArguments {
 	output?: string;
 	/** Delve LoadConfig parameters **/
 	dlvLoadConfig?: LoadConfig;
+	dlvToolPath?: string;
```

Comment:
Any reason to keep this optional?

Reply:
https://github.com/Microsoft/vscode-go/pull/2108#discussion_r232182526